site/en/docs/extensions/reference/history/index.md

```diff
@@ -19,28 +21,52 @@ example:
+`"keyword_generated"`
+: Corresponds to a visit generated for a keyword. <!--How are keyword, keyword_generated, and generated different?-->
```

Comment:
```suggestion
: Corresponds to a visit generated for a keyword.
```

Reply:
I'd like to make sure the user understands the difference, and I don't think what's here is sufficient. What else might readers need to know?

Reply:
I've added keywords to the list.